lib/licensee/rule.rb

```diff
@@ -27,9 +27,10 @@ def all
         end.flatten
       end
 
-      def find_by_tag(tag)
-        Rule.all.find { |r| r.tag == tag }
+      def find_by_tag_and_group(tag, group = nil)
+        Rule.all.find { |r| r.tag == tag && (group.nil? || r.group == group) }
       end
+      alias find_by_tag find_by_tag_and_group
```

Comment:
Yep. Trying to follow http://semver.org/ strictly since https://github.com/benbalter/licensee/pull/100.

This way, if you call `Rule#find_by_tag('foo')` it still works without needing to maintain two different methods.